File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1180.java
import java.util.Scanner;

public class Main {
    public static void main(String args[]) {
        Scanner ler = new Scanner(System.in);
        
        int i, n, v, menor, pos=1;
        n = ler.nextInt();
        menor = ler.nextInt();
        for(i=1;i<n;i++){
            v = ler.nextInt();
            if(v<menor){
                menor = v;
                pos = i;
            }
        }
        System.out.printf("Menor valor: %d\n", menor);
        System.out.printf("Posicao: %d\n", pos);

    }
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1180.cpp
#include <iostream>

using namespace std;

int main(){
    int n, x, menor, pos=0;
    cin >> n;
    cin >> menor;
    for(int i=1;i<n;i++){
        cin >> x;
        if(x<menor){
            menor=x;
            pos=i;
        }
    }
    cout << "Menor valor: " << menor << endl << "Posicao: " << pos << endl;

    return 0;
}